setup.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python
+# This file is part account_es module for Tryton.
+# The COPYRIGHT file at the top level of this repository contains
+# the full copyright notices and license terms.
+
+from setuptools import setup
+import re
+import os
+import io
+try:
+    from configparser import ConfigParser
+except ImportError:
+    from ConfigParser import ConfigParser
+
+MODULE = 'create_transactions'
```

Comment:
It is customary for trytond modules to be named after the functional area or important modules they build upon. In this case I suggest a renaming to account_bank_statement_create_transactions.

The repository name should be renamed accordingly, as well as any URL appearing in the setup.py file